.gitignore

```diff
@@ -13,3 +13,4 @@ dist/
 .ipynb_checkpoints/
 venv/
 my_db_config.yaml
+database.yaml
```

Comment:
you mention this in the summary, i'm just curious where it shakes out? is it something i'm supposed to provide to ... alembic? or to the `triage` command?

Reply:
The results schema readme, which provides instructions for making code changes to the results schema, specifically names `database.yaml` as the expected name for the triage command: https://github.com/dssg/triage/blob/master/src/triage/component/results_schema/README.md

I figured, if we are recommending that people use that filename, it would be good to add it to the .gitignore

Reply:
(not that it _has_ to be stored in the repo or even should be, but better to avoid having some commit such a file, just in case)

Reply:
:thinking: yeah I'm not really sure, especially since I'm not a user and only a sometimes-contributor :smile: 

it's kind of a security issue to encourage people to put that kind of stuff into a file, and there are _a lot_ of easy ways around this for us -- that's why I tend to be skeptical when I see it come up.

but, you're right; until we really figure this out, and so long as there's a risk of committing something like that accidentally, I don't have a problem with it being in the `.gitignore`.